openshift/metrics/README.md

```diff
@@ -0,0 +1,23 @@
+# Usage
+
+1. Make sure you have `oc` installed and targeted the namespace where the sync app is deployed.
+2. Run the script to provision Prometheus and Grafana:
+    ```
+    ./install-metrics.sh
+    ```
+3. Add the following annotation to the `Deployment` or `DeploymentConfig` of your sync apps (change the value for the endpoint to match the actual metrics endpoint if it's not `/metrics`):
```

Comment:
I'm just wondering. I might not get that right. Can we put this into the template and ask people to edit when endpoint is different?